collect/collect_test.go

```diff
@@ -723,3 +723,72 @@ func TestDependencyInjection(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+// TestAddSpanCount tests that adding a root span winds up with a trace object in
+// the cache and that that trace gets span count added to it
+func TestAddSpanCount(t *testing.T) {
```

Comment:
Is it possible to add test cases for late spans?

Reply:
Done.